stories/Components/ScrollContainer/ScrollContainer.jsx

```diff
@@ -0,0 +1,200 @@
+import React, { useRef, useState, useEffect, useCallback } from 'react';
+import PropTypes from 'prop-types';
+import './scroll-container.scss';
```

Comment:
This import should be at `stories\assets\scss\_components.scss`

I'll do this real quick

Reply:
Done in https://github.com/unisdr/undrr-mangrove/pull/219/commits/95046ec3cfc1c1753164b26da9569b071858bae1